invenio_rdm_records/templates/semantic-ui/invenio_notifications/grant-user-access.create.jinja

```diff
@@ -0,0 +1,68 @@
+{% set record = notification.context.record %}
+{% set receiver = notification.context.receiver %}
+{% set message = notification.context.message | safe if notification.context.message else '' %}
+{% set permission = notification.context.permission %}
+
+{% set record_title = record.metadata.title %}
+{% set record_link = record.links.self_html %}
+{% set account_settings_link = "{ui}/account/settings/notifications".format(
+    ui=config.SITE_UI_URL
+    )
+%}
+
+{%- block subject -%}
+{{ _("👤✔️ You were granted {permission} access to record '{record_title}'").format(permission=permission, record_title=record_title) }}
+{%- endblock subject -%}
+
+{%- block html_body -%}
+<table style="font-family:'Lato',Helvetica,Arial,sans-serif;border-spacing:15px">
+    <tr>
+        <td>{{ _("The owner of '{record_title}' has given you permission to {permission} their record").format(record_title=record_title, permission=permission)}}
+            {% if message %}
+            {{  _(" with the following message:")}}
+            {% endif %}
+        </td>
+    </tr>
+    <tr>
+        {% if message %}
+        <td><em>{{message}}</em></td>
+        {% endif %}
+    </tr>
+    <tr>
+        <td><a href="{{ record_link }}" class="button">{{ _("View the record")}}</a></td>
+    </tr>
+    <tr>
+        <td><strong>_</strong></td>
+    </tr>
+    <tr>
+        <td style="font-size:smaller">{{ _("This is an auto-generated message. To manage notifications, visit your")}} <a href="{{account_settings_link}}">{{ _("account settings")}}</a>.</td>
+    </tr>
+</table>
+{%- endblock html_body -%}
+
+{%- block plain_body -%}
+{{ _("The owner of '{record_title}' has given you permission to {permission} their record").format(record_title=record_title, permission=permission)}}
+
+{% if message %}
+{{ _("with the following message:")}}
+{{message}}
+{% endif %}
+
+[{{ _("View the record") }}]({{ record_link }})
+
+{{ _("This is an auto-generated message. To manage notifications, visit your account settings")}}
```

Comment:
Maybe we can link here the notifications settings i.e. `/account/settings/notifications/`

Reply:
I would change the plain text to and remove the MD syntax:
```
View the record: https://...
...visit your account settings: https://...
```